.gitignore

```diff
@@ -93,3 +92,7 @@ test.tap
 # Xcode workspaces and project folders
 *.xcodeproj
 *.xcworkspace
+
+# Visual Studio Code
+.vscode/
+jsconfig.json
```

Comment:
Do we actually want to add that data to .gitignore? If I'm not wrong there is an ongoing discussion in another PR for `.idea` and another open PR to whitelist stuff instead of blacklisting.
I think `.idea`, `.vscode` and the others should be in the user global gitignore not here.